### ISSUE TITLE
Dashboard demos chart scaling

### ISSUE THREAD

**Issue by TomClancyIsDead** [bayvakoof/civstats-app#2]
The demos chart isn't terribly functional at the moment. Using actual values when the categories have different orders of magnitude in their values means that having one of the bigger valued categories makes the others impossible to see.

Potential ideas:
Scale each to % of leader.
Have population and military in thousands to start, and increase the unit scaling to tens of thousands when the values are in the millions.
If there is a single category selected, add lines for average and leader.

**Comment by bayvakoof** [comment]
Agreed, the demographics chart in its current incarnation isn't the best way to visualize the data. But right now i'm not uploading the best/worst demographics (yet), so scaling to % of leader isn't possible.

Actually I really want to migrate to rails5 and make all the data available via JSON for everyone to access but haven't gotten around to it yet.
